python/sglang/srt/server.py

```diff
@@ -578,8 +578,9 @@ def _set_envs_and_config(server_args: ServerArgs):
     os.environ["NCCL_NVLS_ENABLE"] = "0"
     os.environ["TORCH_NCCL_AVOID_RECORD_STREAMS"] = "1"
     os.environ["CUDA_DEVICE_MAX_CONNECTIONS"] = "4"
-    if "GLOO_SOCKET_IFNAME" not in os.environ:
-        os.environ["GLOO_SOCKET_IFNAME"] = "eth0"
+    # TODO(fix socket error with gpu backend)
```

Comment:
Why is this commented out?

Reply:
Is this used for cpu backend or specific workstation? get ```RuntimeError: [enforce fail at pytorch/third_party/gloo/gloo/transport/tcp/device.cc:83] ifa != nullptr. Unable to find address for: eth0```

Reply:
This is used for multi-node tensor parallelism. Instead of using comments, we suggest adding an is_hip flag.